lib/archethic/bootstrap.ex

```diff
@@ -206,12 +208,36 @@ defmodule Archethic.Bootstrap do
       Logger.info("Synchronization finished")
     end
 
+    resync_network_chain([:oracle, :node_shared_secrets])
+
     Sync.publish_end_of_sync()
     SelfRepair.start_scheduler()
-
     :persistent_term.put(:archethic_up, :up)
   end
 
+  def resync_network_chain(type_list) do
+    nodes = P2P.authorized_and_available_nodes()
+
+    Task.Supervisor.async_stream_nolink(Archethic.TaskSupervisor, type_list, fn type ->
+      if addr = TransactionChain.list_addresses_by_type(type) |> Enum.at(0) do
+        with {:ok, last_addr} <- TransactionChain.fetch_last_address_remotely(addr, nodes),
+             {:ok, last_tx} <- TransactionChain.fetch_transaction_remotely(last_addr, nodes),
```

Comment:
maybe you should check if the transaction exists already to avoid the download and this warning

<img width="861" alt="image" src="https://user-images.githubusercontent.com/42943690/185634992-e12cdc93-45de-49e6-b947-611c73b57476.png">

Reply:
In write_transaction DB we are checking for existing txns.
I will redo code with fetch transactions remotely with paging addr